src/Glpi/Form/QuestionType/AbstractQuestionTypeShortAnswer.php

```diff
@@ -103,6 +113,9 @@ class="form-control"
                 placeholder="{{ input_placeholder }}"
                 value="{{ question is not null ? question.fields.default_value : '' }}"
                 aria-label="{{ __('Default value') }}"
+                {% for key, value in attributes %}
+                    {{ key|e }}="{{ value|e('html_attr') }}"
```

Comment:
`|e` is the default filter
```suggestion
                    {{ key }}="{{ value|e('html_attr') }}"
```